docs/self-managed/identity/user-guide/configure-external-identity-provider.md

```diff
@@ -4,8 +4,53 @@ title: "Configuring an external identity provider"
 sidebar_label: "Configuring an external identity provider"
 ---
 
+In this guide we will show you how to configure an external identity provider like OpenID Connect, SAML, LDAP or
+Active Directory.
+
 :::note
-Currently, the Identity UI does not offer support for configuring external identity providers. To configure an external
-identity provider, we recommend referencing the Keycloak documentation for
-[adding a provider](https://www.keycloak.org/docs/16.1/server_admin/index.html#adding-a-provider)
+The Identity UI does not offer support for configuring external identity providers. You can configure an external
```

Comment:
I'd be inclined to move this to the very top of the page so that its clear from the very start that this is something that isn't supported in the Identity UI but instead is achieved via alternative methods

Reply:
Also I wonder if this should be `:::info` https://docusaurus.io/docs/next/markdown-features/admonitions